### ISSUE TITLE
Learning rate in Junior torch model is not picked up

### ISSUE THREAD

**Issue by EricavanderSar** [FraunhoferIEE/CurriculumAgent#8]
The init function of `JuniorPTL(LightningModule)` defines the learning rate with:
`self.lr = model_config.get("learning_rate", 5e-4)`

However this is not used in configure_optimizers():

```
    def configure_optimizers(self):
        """
        Configures the optimizer used for training. Adam optimizer is used with the specified learning rate.

        Returns:
            torch.optim.Optimizer: The optimizer for the model parameters.
        """
        optimizer = torch.optim.Adam(self.parameters(), lr=1e-3)
        return optimizer
```

This can be fixed by adjusting this to:

`optimizer = torch.optim.Adam(self.parameters(), lr=self.lr)`
